### DISCUSSION TITLE
Attribute to LOCK END DATE

### DISCUSSION THREAD

**Discussion by suraj-l** [fetrarij/ngx-daterangepicker-material#483] [Ideas]
There seems to be an attribute to lock start date but there is no way we can lock end date. Why isn't there an attribute to lock end date?

**Comment by suraj-l** [comment]
@fetrarij please help me out either by adding am attribute to lock end date or with a way around to lock the end date once the start date is selected. Thank you!

**Comment by fetrarij** [comment]
Hello @suraj-l , try to specify a maxDate or minDate to lock the endDate

**Reply by suraj-l** [reply]
No @fetrarij I want to lock the end date and not the end of the range. This is similar to the 'lockstartdate' attribute already existing. Just like that I want to lock the end date
